frontend/src/pages/NeuroScan.jsx

```diff
@@ -12,31 +14,43 @@ const useStyles = makeStyles((theme) => ({
       },
     },
   },
+  layoutContainer: {
+    position: 'relative',
+    width: '100%',
+    height: '90vh',
+  },
 }));
 
 export default function NeuroScan() {
   const classes = useStyles();
-
+  const store = useStore();
+  const [LayoutManager, setLayoutManager] = useState(undefined);
```

Comment:
To avoid confusion, I would suggest to rename LayoutManager to sth like "LayoutComponent" or "FlexLayout".

LayoutManager is the Typescript class that contains only the logic while we access here `myManager.getComponent()`